src/lib.rs

```diff
@@ -380,20 +572,21 @@ pub fn delegate(tokens: TokenStream) -> TokenStream {
                 syn::ReturnType::Type(_, ret_type) => {
                     if into {
                         quote::quote! { std::convert::Into::<#ret_type>::into(#body) }
-                    }
-                    else {
+                    } else {
                         body
                     }
                 }
             };
 
-            quote::quote_spanned! {span=>
+            let x = quote::quote_spanned! {span=>
```

Comment:
I suppose that this is some leftover debug code and the expression can be returned directly?

Reply:
Oh, yes. Sorry.